python/lsst/sims/movingObjects/chebyFits.py

```diff
@@ -512,4 +508,4 @@ def write(self, coeffFile, residFile, failedFile, append=False):
         if len(self.failed) > 0:
             with open(failedFile, openMode) as f:
                 for i, failed in enumerate(self.failed):
-                    print >>f, ' '.join([str(x) for x in failed])
\ No newline at end of file
+                    print >>f, ' '.join([str(x) for x in failed])
```

Comment:
I think this needs to be updated to print function standard (with `file=f` as a keyword argument in the function).